src/csharp/Grpc.Core/Grpc.Core.csproj

```diff
@@ -39,18 +39,10 @@
       <PackagePath>runtimes/osx/native/libgrpc_csharp_ext.x64.dylib</PackagePath>
       <Pack>true</Pack>
     </Content>
-    <Content Include="..\nativelibs\csharp_ext_macos_x86\libgrpc_csharp_ext.dylib">
-      <PackagePath>runtimes/osx/native/libgrpc_csharp_ext.x86.dylib</PackagePath>
-      <Pack>true</Pack>
-    </Content>
     <Content Include="..\nativelibs\csharp_ext_linux_x64\libgrpc_csharp_ext.so">
       <PackagePath>runtimes/linux/native/libgrpc_csharp_ext.x64.so</PackagePath>
       <Pack>true</Pack>
     </Content>
-    <Content Include="..\nativelibs\csharp_ext_linux_x86\libgrpc_csharp_ext.so">
```

Comment:
Am I right in saying these x86 versions won't appear in *any* package in 2.34.0, if this goes ahead? Presumably we can create another package for the Mac/Linux x86 versions if they turn out to be needed?

Reply:
yes, x86 versions for linux and macos will just go away. We could have a separate package, but coincidentally, we might lose the ability to build the x86 macos artifact on our CI soon anyway (the OS X version that our CI kokoro provides doesn't allow you to build x86 binaries anymore, unless you explicitly opt in to using out of deprecated mac images) and removing x86 mac artifact altogether would make that problem go away, so I opted for simplicity here and decided to just remove the artifacts. 

More on removal of x86 linux and macos artifacts is here: https://github.com/grpc/grpc/pull/24378

Reply:
And yes, we could add a separate package for Mac/Linux x86 if the demand for them is high (which I don't expect).